api/routes/user.py

```diff
@@ -512,3 +514,52 @@ def delete_favorite_experts(
     db.refresh(current_user)
 
     log(log.INFO, "User [%s] successfully updated favorite experts list", current_user.id)
+
+
+@user_router.put(
+    "/avatar",
+    status_code=status.HTTP_200_OK,
+    response_model=s.UserPut,
+    responses={
+        status.HTTP_400_BAD_REQUEST: {"description": "Unknown file extension"},
+    },
+    dependencies=[Depends(get_current_user)],
+)
+def upload_user_avatar(
+    file: UploadFile,
+    db: Session = Depends(get_db),
+    s3_client: S3Client = Depends(get_s3_connect),
+    current_user: m.User = Depends(get_current_user),
+):
+    """Uploads file for user avatar"""
+
+    extension = get_file_extension(file)
+
+    file_type = c.get_file_type(extension)
+
+    if file_type == s.FileType.UNKNOWN:
+        log(log.ERROR, "Unknown file extension [%s]", extension)
+        raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST, detail="Unknown file extension")
+
+    file_model = c.create_file(
+        file=file,
+        db=db,
+        s3_client=s3_client,
+        extension=extension,
+        file_type=file_type,
+        file_name_url="user/avatar",

```

Comment:
Ate yo sure this would not override avatar of one user by avatar of another user?

Reply:
May be do smth like f"users/{user.uuid}/avatar"
?